snap/snapcraft.yaml

```diff
@@ -177,6 +185,7 @@ parts:
       - git
       - grep
       - hostname
+      - ufw
```

Comment:
Why do we include ufw in the snap?

Reply:
We are setting the python path environment variable globally in the snap to use the one we ship which is Python 3.8. ufw uses Python, and the snap uses the hosts ufw binary on classic which can lead to errors due to python version mismatch. For example the ufw on 22.04 doesn't seem to be compatible with Python3.8.

So the idea is to include the ufw so we can avoid using the host binary. I'm open to suggestions on how we can handle this.